server/static_content_macro.cc

```diff
@@ -216,7 +226,7 @@ writeInternalLink(Utf8String url,
     }
 
     writeHtml("<a href=\"");
-    writeText(url);
+    writeText(patchUrl(url));
```

Comment:
can you add this method to the `handleLink` function we were initially looking at as well plz?